File: app/src/main/java/com/github/nikartm/stripedprocessbutton/MainActivity.java
package com.github.nikartm.stripedprocessbutton;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.widget.Button;

import com.github.nikartm.support.StripedProcessButton;


public class MainActivity extends AppCompatActivity {

    private StripedProcessButton stripedButton;
    private Button btnStop;
    private StripedProcessButton btn_2;
    private StripedProcessButton btn_3;
    private StripedProcessButton btn_4;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        stripedButton = findViewById(R.id.btn_start);
        btnStop = findViewById(R.id.btn_stop);
        btn_2 = findViewById(R.id.btn_2);
        btn_3 = findViewById(R.id.btn_3);
        btn_4 = findViewById(R.id.btn_4);

        stripedButton.setOnClickListener(v -> stripedButton.start());
        btnStop.setOnClickListener(v -> stripedButton.stop());

//        initStripedButton();

//        stripedButton.setStripeWidth(13).setStripeGradient(false);
//        stripedButton.start();

        btn_2.start();
        btn_3.start();
        btn_4.start();
    }

    private void initStripedButton() {
        stripedButton.setCornerRadius(50)
                .setStripeAlpha(0.7f)
                .setStripeRevert(true)
                .setStripeGradient(false)
                .setTilt(15)
                .start();
    }
}

File: app/src/main/java/com/github/nikartm/stripedprocessbutton/MainActivity.kt
package com.github.nikartm.stripedprocessbutton

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.github.nikartm.stripedprocessbutton.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        bindView()
    }

    private fun bindView() = with(binding) {
        initStartStripedButton()
        btnStart.setOnClickListener {
            btnStart.start()
            btn2.start()
            btn3.start()
            btn4.start()
        }
        btnStop.setOnClickListener {
            btnStart.stop()
            btn2.stop()
            btn3.stop()
            btn4.stop()
        }
    }

    private fun initStartStripedButton() = with(binding.btnStart) {
        text = "Start process"
        setCornerRadius(50f)
        setStripeAlpha(0.7f)
        setStripeRevert(true)
        setStripeGradient(false)
        setTilt(15f)
    }
}
